deploy/clowdapp.yaml

```diff
@@ -73,7 +73,7 @@ objects:
         name: provisioning-backend
         version: 13
       dependencies:
-        - sources-api
+        - sources-api-go
```

Comment:
this does not work for me locally. but using `bonfire deploy sources provisioning` I get the sources-api backed by the go code.

Reply:
I've updated the docs here: https://github.com/RHEnVision/provisioning-backend/pull/78/files ... you can use appsre and it still overrides the locally defined apps :)

Reply:
that's still reading the service from `deploy/bonfire.yaml` config file, its not being read from appsre itself.